### PR TITLE
text change

### LINKED ISSUE #27: Elaborate on binary pls
I thought I understood binary somewhat but in 1.4 binary and hexadecimal you said twice that 10_0000_0000 is 2048 but 0>2>4>8>16>32>64>128>256>512 is what I get.

If I slap the number in a calc or google I get 512 as well.

### PR DESCRIPTION
**Pull request by return2monki**
I didn't build mdbook here but the link should work correctly. One thing I noticed is that the next hexadecimal section makes a little less sense now as I intentionally left out the underscores after mentioning them. They could just be added anyway, I don't think it matters too much.

Fixes #27.

### CODE REVIEW

**Review by return2monki — COMMENTED** [review]
changed to 1028 (didn't see this was asked for initially)

File: src/part1/bin_and_hex.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Thus, we can generalize the principle outlined above, and write these two number
        ↑          ↑         ↑         ↑         ↑         ↑         ↑         ↑         ↑         ↑         ↑
 ```
 
-So, by applying the same principle, we can say that in base 2, 42 is written as `101010`, and 2048 as `1000000000`.
+So, by applying the same principle, we can say that in base 2, 42 is written as `10_1010`, and 1028 as `100_0000_0000`. Underscores are used here to improve readability[^underscores].
```
Comment (ISSOtm) [inline src/part1/bin_and_hex.md:49]:
```suggestion
So, by applying the same principle, we can say that in base 2, 42 is written as `101010`, and 2048 as `10000000000`. 
```

File: src/part1/bin_and_hex.md [hunk 2]
```diff
@@ -113,4 +113,5 @@ If you're having trouble converting between decimal and binary/hexadecimal, chec
 - In RGBDS assembly, the hexadecimal prefix is `$`, and the binary prefix is `%`.
 - Hexadecimal can be used as a "compact binary" notation.
 - Using binary or hexadecimal is useful when individual bits matter; otherwise, decimal works just as well.
+[^underscores]:
```
Comment (ISSOtm) [inline src/part1/bin_and_hex.md:116]:
```suggestion
[^underscores]:
```

**Review by ISSOtm — CHANGES_REQUESTED** [review]
Underscores are intentionally not used so as to drive the point home better.

File: src/part1/bin_and_hex.md (hunk 1 above)
Comment (return2monki) [reply src/part1/bin_and_hex.md:49]: Understandable, 1 more 0 has to be added if you want it to be 2048 right?

Comment (ISSOtm) [reply src/part1/bin_and_hex.md:49]: Yes, but I want there to be a mistake—to make the reader *experience* why it's inconvenient.

**Comment by return2monki** [comment]
Ok that's it, I didn't realize the first binary number was intentionally wrong.

**Comment by ISSOtm** [comment]
Ah, only the latter (under "hexadecimal") is supposed to. The block just above also specifies 1024 instead of 2048, so it's likely a copy-paste error; *that* one should be made right.

**Comment by return2monki** [comment]
> Ah, only the latter (under "hexadecimal") is supposed to. The block just above also specifies 1024 instead of 2048, so it's likely a copy-paste error; _this_ one should be made right.

I made it 1024 myself, before all this it was 2048 with 10 digits. Do you want 1024 or 2048 with correct binary?

**Comment by ISSOtm** [comment]
It should be 1024 for consistency with the preformatted block above.

**Review by ISSOtm — APPROVED** [review]

**Pull request merged**